.github/ISSUE_TEMPLATE.md

```diff
@@ -10,31 +15,38 @@
 2. ...
 3. ...
 
-<!-- Please attach (or link to) configuration files if applicable -->
+<!--
+Please upload relevant configuration (as .txt).
+If you use the command "ttn-lw-stack config", you can redact sensitive config.
+-->
 
-**What do you see now?**
-<!-- Please attach (or link to) screenshots and logs if applicable -->
+**What is already there? What do you see now?**
+<!--
+Please paste terminal output, upload logs (as .txt) or upload screenshots.
+Describe or link to related APIs, screen designs, packages, etc.
+-->
 
 ...
 
-**What do you want to see instead?**
-<!-- Please add some examples or mock-ups if applicable -->
+**What is missing? What do you want to see?**
+<!-- Please add some examples or mock-ups if applicable. -->
 
 ...
 
-**How do you propose to implement this?**
+**Environment:**
 <!--
-If unsure, add the discussion label and (temporarily) assign the expert
-If you cannot assign people, please @mention the experts
+Your environment: OS/Browser/Gateway/Device/...? Versions? IDs/EUIs?
+Paste the output of "ttn-lw-cli version" or "ttn-lw-stack version" if applicable.
 -->
 
 ...
 
-**Environment:**
-<!-- Your environment: OS/Browser/Gateway/Device/...? Versions? IDs/EUIs? -->
+**How do you propose to implement this?**
+<!-- Please think about how this could be implemented. -->
 
 ...
 
 **What can you do yourself and what do you need help with?**
```

Comment:
I don't like this formulation, as it is two contrary questions in one and which one is consequently answered is sometimes hard to distinguish. A solution would be to either get rid of one of the question or to split it up into two questions.

Reply:
I personally think it's fine. I typically put things like "Can do" or "Review and test" here. Note that this issue template is primarily aimed at community contributors.

Reply:
I think @kschiffer has a good point, the question is indeed a bit confusing.

How about the following:

```
**Can you do this yourself and submit a Pull Request?**
<!-- You can also @mention experts if you need help with this. -->
```

Then @johanstokking can write something like "No, but I can review after @htdvisser implements it". At the same time it encourages external contributors to consider submitting a pull request, and we let them know that they can ask for help.